sentry-logback/src/test/kotlin/io/sentry/logback/SentryAppenderTest.kt

```diff
@@ -0,0 +1,148 @@
+package io.sentry.logback
+
+import com.nhaarman.mockitokotlin2.any
+import com.nhaarman.mockitokotlin2.argumentCaptor
+import com.nhaarman.mockitokotlin2.eq
+import com.nhaarman.mockitokotlin2.mock
+import com.nhaarman.mockitokotlin2.verify
+import io.sentry.core.ISentryClient
+import io.sentry.core.Sentry
+import io.sentry.core.SentryEvent
+import io.sentry.core.SentryLevel
+import kotlin.test.Ignore
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+
+class SentryAppenderTest {
+    private class Fixture {
+        val client = mock<ISentryClient>()
+        val captor = argumentCaptor<SentryEvent>()
```

Comment:
not really against using `argumentCaptor`, just out of curiosity, is there any advantage of using it or the normal `argWhere`, `argThat`, `argForWhich` or even `check`? they look more natural to me when asserting values of a `verify` call.

https://github.com/nhaarman/mockito-kotlin/wiki/Mocking-and-verifying

Reply:
Thanks for hint, I missed the existence of `check`